grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -685,6 +701,7 @@ public GrpcService build() {
                 unsafeWrapRequestBuffers,
                 useClientTimeoutHeader,
                 maxInboundMessageSizeBytes);
-        return enableUnframedRequests ? new UnframedGrpcService(grpcService, handlerRegistry) : grpcService;
+        return enableUnframedRequests ? new UnframedGrpcService(grpcService, handlerRegistry,
```

Comment:
As you know `unframedErrorResponseMapper` is vaild only when unframed gRPC is enabled.
How about raisng an `IllegalStateException` if unframed gRPC is disabled.